docker/build.sh

```diff
@@ -214,6 +206,10 @@ function docker_cmd()
             $DOCKER_REMOTES_BINARY tag $IMAGE:${DOCKER_TAG} $IMAGE:${GIT_BRANCH}
             $DOCKER_REMOTES_BINARY push $IMAGE:${GIT_BRANCH}
 
+            echo "pushing $GCR_IMAGE docker image..."
+            $DOCKER_REMOTES_BINARY tag $IMAGE:${DOCKER_TAG} ${GCR_IMAGE}:${DOCKER_TAG}
+            $GCR_REMOTES_BINARY push ${GCR_IMAGE}:${DOCKER_TAG}
```

Comment:
For dockerhub we push `<image>:<git-sha>` _and_ `<image>:<git-branch>`. For GCR we only push `<image>:<git-sha>`. Do we need to push a tag with the branch name in GCR as well?

Reply:
we shouldn't, because as of now GCR is only for deploys, which use the hash-tagged images.  branch-tagged images are for fiabs, which still pull from dockerhub only.